### DISCUSSION TITLE
Found a new way to get around YouTube preventing you from logging in

### DISCUSSION THREAD

**Discussion by zewowo** [qnblackcat/uYouPlus#770] [General]
I don't know if anyone had said this before but heres how you can do it:
Quick note: THIS IS ONLY FOR PEOPLE USING SIDELOADLY (I DON'T THINK YOU CAN DO IT USING ALTSTORE)

1. You have to use [Sideloadly](https://sideloadly.io/)

3. drag the IPA on to [Sideloadly](https://sideloadly.io/)

5. BEFORE INSTALLING, make sure these two options are enabled: "Inject dylibs/frameworks" and also "Sideload Spoofer". Make sure you only have those two checked in the Tweak Injection section.

7. Press the start button

9. Login to your icloud account

11. Profit

**Comment by johnedge7733** [comment]
Worked!!!

**Reply by IlyMooon** [reply]
Worked too, since i got lots of trouble while running the "official' workaround, its saved it.
